src/InitialValues.jl

```diff
@@ -452,4 +449,42 @@ asmonoid(op) = hasinitialvalue(op) ? op : AdjoinIdentity(op)
                        x::SpecificInitialValue{AdjoinIdentity{OP}}) where OP = x
 hasinitialvalue(::Type{AdjoinIdentity{T}}) where {T} = true
 
+"""
+    adjoininit(rf) -> rf′
+
+Transform a binary function `rf` (aka reducing function) to a function
+`rf′` that has an initial value; i.e., `rf′(Init(rf′), x) === x` and
+`hasinitialvalue(rf′) === true`.  Do nothing if `rf` already has a
+known initial value.
```

Comment:
This API is not possible unless `Transducers.start` is called via `Init(rf′)`. See:

https://github.com/tkf/Transducers.jl/pull/289/commits/6e2f277dd33bdb3a69ba7462211220bd4e65f7c1
https://travis-ci.com/github/tkf/Transducers.jl/jobs/339334491#L338